aspnet/fundamentals/caching.rst

```diff
@@ -1,8 +1,157 @@
-.. include:: /../common/stub-topic.txt
+In Memory Caching
+=================
 
-|stub-icon| Caching
-===================
+By `Steve Smith`_
 
-.. include:: /../common/stub-notice.txt
+Caching involves keeping a copy of data in a location that can be accessed more quickly than the source 
+data. ASP.NET 5 has rich support for caching in a variety of ways, including keeping data in memory on the 
+local server, which is referred to as *in memory caching*.
 
-.. _issue: https://github.com/aspnet/Docs/issues/69
\ No newline at end of file
+In this article:
+	- `Caching Basics`_
+	- `Configuring In Memory Caching`_
+	- `Reading and Writing to a Memory Cache`_
+	- `Cache Dependencies and Callbacks`_
+
+`View or download sample from GitHub <https://github.com/aspnet/Docs/tree/1.0.0-beta8/aspnet/fundamentals/caching/sample>`_.
+
+Caching Basics
+--------------
+Caching can dramatically improve the performance and scalability of ASP.NET applications, by eliminating unnecessary requests to external data sources for data that hasn't changed since it was last requested. ASP.NET supports several different kinds of caches, the simplest of which is represented by the `IMemoryCache <https://github.com/aspnet/Caching/blob/dev/src/Microsoft.Extensions.Caching.Abstractions/IMemoryCache.cs>`_ interface, which represents a cache stored in the memory of the local web server.
```

Comment:
"for data that hasn't changed since it was last requested" -> "for data that changes less frequently than it is read"